qt/res/OrganicMaps.desktop

```diff
@@ -1,13 +1,13 @@
 [Desktop Entry]
 Type=Application
-Name=MapsWithMe
+Name=Organic Maps
 Version=1.0
 GenericName=Detailed Offline Maps of the World
 GenericName[ru]=Подробная оффлайновая карта мира
-Icon=/usr/share/MapsWithMe/logo.png
-TryExec=/usr/bin/MapsWithMe
-Exec=/usr/bin/MapsWithMe
+Icon=organicmaps
+TryExec=OMaps
```

Comment:
I think we should rename binary too.

Reply:
If you do please mention it in the release notes so that I know when to modify the nixos package